client/app/services/intercom.js

```diff
@@ -1,13 +1,18 @@
 import Ember from 'ember';
 import config from 'client/config/environment';
 
+/* eslint-disable new-cap*/
 export default Ember.Service.extend({
-  intercom: window.Intercom || Ember.K,
+  init() {
+    this._super(...arguments);
+    if (config.intercom.disabled) {
+      window.Intercom = Ember.K;
```

Comment:
Also, instead of checking a new `disabled` property we introduce, could we just check the `appId` property that's already there (or not)?